diagnostic reports/README.md

```diff
@@ -0,0 +1,374 @@
+  <specimenRole classCode="SPEC">
+   <id root="DB2E5F7D-B0D0-488A-825A-BE58D09953CF"/>
+   <id extension="C3179539H" root="2.16.840.1.113883.2.1.4.5.2"/>
+   <effectiveTime>
+    <center value="20030627000000"/>
+   </effectiveTime>
+   <specimenSpecimenMaterial classCode="MAT" determinerCode="INSTANCE">
+    <desc>Plasma</desc>
+   </specimenSpecimenMaterial>
+  </specimenRole>
+ </specimen>
+ <component contextConductionInd="true" typeCode="COMP">
+  <NarrativeStatement classCode="OBS" moodCode="EVN">
+   <id root="DF0FEFF0-32C1-4BB9-8DDD-76E2D13B5D0F"/>
+   <text mediaType="text/x-h7uk-pmip">CommentType:LAB SPECIMEN COMMENT(E271)
+CommentDate:20030627000000
+
+Quantity: 1750.000 mL
+Received Date: 2003-06-27 18:24
+   </text>
+   <statusCode code="COMPLETE"/>
+   <availabilityTime value="20030628112300"/>
+  </NarrativeStatement>
+ </component>
+ <component contextConductionInd="true" typeCode="COMP">
+  <CompoundStatement classCode="BATTERY" moodCode="EVN">
+   
+   ...
+   
+  </CompoundStatement>
+ </component>
+</CompoundStatement>
+```
+</details>
+
+5. Where `extension[index].url` equals `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid"`, 
+`extension[index2].url` equals `"descriptionDisplay"` and `extension[index2].value` does not equal `Specimen.type.coding[0].display` 
+
+### Specimen Level Narrative Statements
+
+A Narrative Statement is created to preserve data from FHIR fields that have no direct mapping.
+
+| Mapped to (XML HL7 NarrativeStatement element) | Mapped from (JSON FHIR / other source ) |
+|------------------------------------------------|-----------------------------------------|
+| id [@root]                                     | Unique ID generated by the adaptor      |
+| text [@mediaType]                              | fixed value = `"text/x-h7uk-pmip"`      |
+| text                                           | EDIFACT comment, as described below     |
+| statusCode                                     | fixedValue = `"COMPLETE"`               |
+| availabilityTime [@value]                      | `DiagnosticReport.issued`               |
+
+#### Text Element Mapping
+
+Unlike a diagnostic report `CompoundStatement`, which can have multiple `NarrativeStatement` components, the Specimen 
+`CompoundStatement` has a single `NarrativeStatement`. The Specimen fields without a direct mapping are concatenated and
+seperated by newlines, before being inserted into the `text` element of the `NarrativeStatement` as an EDIFACT comment 
+(example in the XML above). The comment type of the EDIFACT comment is fixed to `LAB SPECIMEN COMMENT(E271)`.
+
+The fields / values mapped to the comment are as follows:
+
+- fixed value = `"EMPTY SPECIMEN"` if no Observations referenced in `DiagnosticReport.result` reference the `Specimen`.  
+- `Specimen.receivedTime` prepended with `"Received Date:"`
+- `Specimen.collection.extention[index].valueCodeableConcept` prepended with `"Fasting Status:"`
```

Comment:
Worth mentioning that it's looking for an extension with URL "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-FastingStatus-1"?

Reply:
yes, added